### PR TITLE
Makefile: set the url of chart repository

### PR DESCRIPTION
**Pull request by hectorj2f**
We need to set the url of the chart repository to get the index with the appropriate entry.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -50,8 +50,7 @@ $(STABLE_TARGETS) $(STAGING_TARGETS): $(TMPDIR)/.helm/repository/local/index.yam
 %/index.yaml: $(STABLE_TARGETS) $(STAGING_TARGETS)
 %/index.yaml: $(TMPDIR)/.helm/repository/local/index.yaml
 	@mkdir -p $(patsubst %/index.yaml,%,$@)
-	$(HELM) --home $(TMPDIR)/.helm repo index $(patsubst %/index.yaml,%,$@)
+	$(HELM) --home $(TMPDIR)/.helm repo index $(patsubst %/index.yaml,%,$@) --url=https://mesosphere.github.io/charts/$(shell echo $@ | sed 's@docs/\(.*\)index.yaml@\1@')
```
Comment (joejulian) [inline Makefile]:
```suggestion
	$(HELM) --home $(TMPDIR)/.helm repo index $(patsubst %/index.yaml,%,$@) --url=https://mesosphere.github.io/charts/$(patsubst docs/%index.yaml,%,$@)
```

File: Makefile [hunk 2]
```diff
@@ -21,10 +21,22 @@ clean:
...
+.PHONY: publish-ci
+publish-ci:
+	@git remote add publish git@github.com:mesosphere/charts >/dev/null 2>&1 || true
+	@git branch -d master >/dev/null 2>&1 || true
+	@git checkout -B master
+	@make all
+	@git add .
+	@git commit -m 'docs: publish repo updates'
+	@git push -f publish $(PULL_REQUEST_ID)
+	@git request-pull master ./
```
Comment (hectorj2f) [inline Makefile]: the final intention would be to open a PR.

**Comment by hectorj2f** [comment]
Once we merge this PR, we need to rebase `dev` against `master` to get this change too.

File: Makefile (hunk 2 above)
Comment (joejulian) [reply Makefile]: I don't want a PR, I want a push. The review was already done for the dev branch. I would call master anything else but github pages will only publish from a branch named "master".

Comment (joejulian) [reply Makefile]: That's why all PR's are supposed to be against dev.

**Comment by joejulian** [comment]
Need to be a PR to `dev`. Master is for github pages publication only.

**Pull request closed without merging**